HomeWork3/Task2/Program.cs

```diff
@@ -0,0 +1,214 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+internal class Massive
+{
+    public static void Main(string[] args)
+    {
+        string str;
+        bool VariableForTryParse;
+        int timeVariable2 = 0,
```

Comment:
it is better to create variables directly in places where you are going to use them